stream/src/connection/instance.rs

```diff
@@ -12,7 +11,6 @@ use chacha20poly1305::{
 use commonware_cryptography::{PublicKey, Scheme};
 use commonware_macros::select;
 use commonware_runtime::{Clock, Sink, Spawner, Stream};
-use prost::Message;
 use rand::{CryptoRng, Rng};
 
 const CHUNK_PADDING: usize = 64 /* protobuf overhead */ + 12 /* chunk info */ + 16 /* encryption tag */;
```

Comment:
chunks are something implemented by `p2p` (not stream), we should think about how to best handle this const.

Reply:
Fixed by https://github.com/commonwarexyz/monorepo/issues/185